### PR TITLE
Select line style hook

### PR DESCRIPTION
**Pull request by NullMad**

### CODE REVIEW

File: src/components/facet/facetVertical.js [hunk 1]
```diff
@@ -453,6 +453,7 @@ FacetVertical.prototype._updateSparkline = function() {
 			selectedSparklinePath.appendTo(sparkline);
 
 			totalSparklinePath[0].classList.add('facet-sparkline-total');
+            totalSparklinePath[0].classList.add('facetkey-sparkline-total');
```
Comment (darionco) [inline src/components/facet/facetVertical.js]: is this a change specific to facet key?
if so, how would that affect other projects using this component?

Comment (NullMad) [inline src/components/facet/facetVertical.js]: This merely adds a unique class that FacetKey can override using a precise bit of CSS. If this class is undefined in other projects it should do nothing.

Comment (darionco) [inline src/components/facet/facetVertical.js]: Cool, thanks for the clarification.

**Comment by jryu01** [comment]
This is no longer relevant.

**Pull request closed without merging**